client-runtime/src/main/java/com/microsoft/rest/RestException.java

```diff
@@ -16,7 +17,12 @@
     /**
      * Information about the associated HTTP response.
      */
-    private Response<ResponseBody> response;
+    private Response<ResponseBody> responseOkHttp;
```

Comment:
Smells like something that's too entrenched to factor out right away?

Reply:
Correct. I tried to remove this, but then I started getting errors in azure-client-runtime. I don't want to start messing with that package quite yet, so I'm leaving this here for now.